finetuner-workflow/finetuner/finetuner.py

```diff
@@ -436,7 +443,7 @@ def main_process_print(*args, **kwargs):
     if "eos_token" not in tokenizer.special_tokens_map:
         tokens_to_add["eos_token"] = "<|endoftext|>"
     if "pad_token" not in tokenizer.special_tokens_map:
-        tokens_to_add["pad_token"] = "<|endoftext|>"
+        tokens_to_add["pad_token"] = tokenizer.eos_token
```

Comment:
This needs to match `gpt_bpe`'s hardcoded default (which should currently be unsigned -1).